src/include/duckdb/storage/data_table.hpp

```diff
@@ -46,6 +44,8 @@ struct ConstraintState;
 struct TableUpdateState;
 enum class VerifyExistenceType : uint8_t;
 
+typedef vector<reference<const ColumnDefinition>> column_defs_t;
```

Comment:
Let's turn this either into a `ColumnList`, or (maybe preferrably) just use indexes into the tables' `ColumnList`